objectivec/GPBMessage_PackagePrivate.h

```diff
@@ -42,6 +42,103 @@
 
 #import "GPBBootstrap.h"
 
+#if defined(__LP64__) && __LP64__
+#define GPB_ASM_PTR " .quad "
+#define GPB_ASM_PTRSIZE " 8 "
+#define GTP_ASM_ONLY_LP64(x) x
+#define GTP_ALIGN_SIZE " 3 "
+#else  // __LP64__
+#define GPB_ASM_PTR " .long "
+#define GPB_ASM_PTRSIZE " 4 "
+#define GTP_ASM_ONLY_LP64(x)
+#define GTP_ALIGN_SIZE " 2 "
+#endif  // __LP64__
+
+#define GPB_MESSAGE_CLASS_NAME_RAW GPBMessage
+#define GPB_STRINGIFY_INTERNAL(x) #x
+#define GPB_STRINGIFY(x) GPB_STRINGIFY_INTERNAL(x)
+#define GPB_MESSAGE_CLASS_NAME GPB_STRINGIFY(GPB_MESSAGE_CLASS_NAME_RAW)
+
+// This generates code
```

Comment:
Likely needs a more completely comment about what all this does and why.

Is there anything we can do detect if this will break in a future tools seed?